go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	golang.org/x/sync v0.0.0-20201207232520-09787c993a3a
 	golang.org/x/sys v0.0.0-20210324051608-47abb6519492
 	google.golang.org/grpc v1.33.2
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
We already import `github.com/pelletier/go-toml`, can we use that one instead?

Reply:
I can agree. I was on the fence since Kubernetes is pretty much YAML and the change is on CRI.

Reply:
Done